public/vendor/plugins/sortt/sortt.js

```diff
@@ -0,0 +1,61 @@
+/*
+  Sortt
+
+  @License MIT
+  @Author 6543
+*/
+
+//creat global function with main routine
+window.sortt=function(normsort,revsort,isdefault){
+  //sortTable [normsort] (revsort) (isdefault)
+
+  //normsort is needet
+  if (typeof normsort  === 'undefined') return false;
+
+  //default values of optinal parameters
+  if (typeof revsort  === 'undefined') revsort = "";
+  if (typeof isdefault  === 'undefined') isdefault = false;
+  if (!(typeof isdefault  === 'boolean')) isdefault = false;
+
+  // parse URL
+  /*  script check if url has already a sort=
+        NO: if sort=“” indikates default (with * prefix) set url_sort=
+        YES: set url_sort variable
+  */
+  url_sort="";
+  if (window.location.search.indexOf("sort=") < 0) {
+    if ( isdefault ) url_sort = normsort;
+  } else {
+    url_sort=window.location.search.substr(window.location.search.indexOf("sort=")+5);
```

Comment:
can't we use URL object to get more pretty usage like this: https://stackoverflow.com/questions/979975/how-to-get-the-value-from-the-get-parameters ?

Reply:
shure :D